### PR TITLE
Ajenner/ui

### PR DESCRIPTION
**Pull request by ajenner**
Updated changelog to reflect previous changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file.
   separate documents.
 - Updated user interface with loading icon to provide visual representation of
   the request's duration.
+- Changed the homepage to update when a file is selected not when it is analysed, based on client feedback.
+- Increased the response timeout delay from 5 seconds to 15 seconds.
```
Comment (22a) [inline CHANGELOG.md]: i don't think this should be here, it's more of an implementation detail than a user facing change

**Comment by 22a** [comment]
confused by what's going on with this pr, heavy squash needed

File: CHANGELOG.md (hunk 1 above)
Comment (c-brenn) [reply CHANGELOG.md]: Worth mentioning, but perhaps reworded - 'fixed bug where analysis requests would time out...'

**Review by c-brenn — CHANGES_REQUESTED** [review]
see comments

**Review by c-brenn — APPROVED** [review]

**Pull request merged**